spec/models/product_spec.rb

```diff
@@ -1,509 +1,557 @@
 require "rails_helper"
 
 RSpec.describe Product do
-  subject(:product) { create(:instrument_requiring_approval) }
+  describe "with everything configured" do
```

Comment:
Most of the changes in this file are me indenting all the specs inside a new `describe` block because I don't need all the setup in my new specs. Github isn't catching that very well.